src/ray/common/scheduling/cluster_resource_scheduler.cc

```diff
@@ -162,7 +162,7 @@ bool NodeResources::operator==(const NodeResources &other) {
   return true;
 }
 
-std::string NodeResources::DebugString() const {
+std::string NodeResources::DebugString(StringIdMap string_to_in_map) const {
```

Comment:
Is this supposed to be `string_to_id_map`? Otherwise not sure what it means.

Reply:
Yes, this argument is the `string_id_map_` in `ClusterResourceScheduler`, and keeps the mapping between the id and the name (string) of each custom resource. Ids are used to improve the performance; it's much cheaper to compare integers than strings. 

Initially, `DebugString` was a method of `ClusterResourceScheduler` but Stephanie convinced me to move it out, so now we need to pass `string_to_id_map_` as an argument.